tests/nwis_test.py

```diff
@@ -5,84 +5,85 @@
 import pandas as pd
 import pytest
 
-from dataretrieval.nwis import NWIS_Metadata
-from dataretrieval.nwis import get_info, get_record, preformat_peaks_response, get_iv, what_sites
+from dataretrieval.nwis import (
+    NWIS_Metadata,
```

Comment:
I've never seen this syntax before. Interesting.

Reply:
yup, this is correct. I would trust the linter-formatter for most anything except when it comes to line breaks.